src/ring_theory/power_series/basic.lean

```diff
@@ -738,6 +745,39 @@ protected lemma inv_eq_iff_mul_eq_one {φ ψ : mv_power_series σ k} (h : consta
   ψ⁻¹ = φ ↔ φ * ψ = 1 :=
 by rw [eq_comm, mv_power_series.eq_inv_iff_mul_eq_one h]
 
+@[simp] protected lemma inv_mul (φ ψ : mv_power_series σ k) :
+  (φ * ψ)⁻¹ = ψ⁻¹ * φ⁻¹ :=
+begin
+  by_cases h : constant_coeff σ k (φ * ψ) = 0,
+  { rw inv_eq_zero.mpr h,
+    simp only [map_mul, mul_eq_zero] at h,
+    -- we don't have `no_zero_divisors (mw_power_series σ k)` yet,
+    cases h;
+    simp [inv_eq_zero.mpr h] },
+  { rw [mv_power_series.inv_eq_iff_mul_eq_one h],
+    simp only [not_or_distrib, map_mul, mul_eq_zero] at h,
+    rw [←mul_assoc, mul_assoc _⁻¹, mv_power_series.inv_mul_cancel _ h.left, mul_one,
+        mv_power_series.inv_mul_cancel _ h.right] }
+end
+
+@[simp] lemma inv_one : (1 : mv_power_series σ k)⁻¹ = 1 :=
+by { rw [mv_power_series.inv_eq_iff_mul_eq_one, mul_one], simp }
+
+@[simp] lemma inv_C (r : k) : (C σ k r)⁻¹ = C σ k r⁻¹ :=
```

Comment:
I think with postfix notation we usually also do postfix naming.
```suggestion
@[simp] lemma C_inv (r : k) : (C σ k r)⁻¹ = C σ k r⁻¹ :=
```

Reply:
```
(base) ➜  mathlib git:(master) ✗ rg -w zero_inv | rg lemma
(base) ➜  mathlib git:(master) ✗ rg -w inv_zero | rg lemma
src/algebra/group/with_one.lean:@[simp] lemma inv_zero [has_inv α] :
src/ring_theory/dedekind_domain.lean:lemma inv_zero' : (0 : fractional_ideal R₁⁰ K)⁻¹ = 0 := fractional_ideal.div_zero
src/algebra/group_with_zero/defs.lean:@[simp] lemma inv_zero : (0 : G₀)⁻¹ = 0 :=
src/linear_algebra/matrix/nonsingular_inverse.lean:@[simp] lemma inv_zero : (0 : matrix n n α)⁻¹ = 0 :=
src/data/real/ennreal.lean:@[simp] lemma inv_zero : (0 : ℝ≥0∞)⁻¹ = ∞ :=
src/data/complex/basic.lean:protected lemma inv_zero : (0⁻¹ : ℂ) = 0 :=
src/data/zmod/basic.lean:lemma inv_zero : ∀ (n : ℕ), (0 : zmod n)⁻¹ = 0
src/data/complex/is_R_or_C.lean:protected lemma inv_zero : (0⁻¹ : K) = 0 :=
```
```
for token in one zero smul; do echo $token; rg -w inv_${token} | rg "(lemma|theorem)" -c; rg -w ${token}_inv | rg "(lemma|theorem)" -c || echo 0; done
one
5
4
zero
9
0
smul
3
5
```